src/oas2/transformers/params.ts

```diff
@@ -210,5 +210,9 @@ function buildSchemaForParameter(
     schema.minLength = 1;
   }
 
-  return { schema, description: param.description };
+  return {
+    schema,
+    description: param.description,
+    ...('x-deprecated' in param && { deprecated: param['x-deprecated'] }),
```

Comment:
Can you add a test for this? 😁

 https://github.com/stoplightio/http-spec/blob/master/src/oas2/transformers/__tests__/params.test.ts

Reply:
Done